src/controllers/userController.js

```diff
@@ -181,10 +169,14 @@ userController.get('/getMyEvents', async (req, res) => {
     }
 });
 
-userController.get('/getMyFavourites', async (req, res) => {
+// Approving / Unapproving user/organizer
```

Comment:
here is "unapproving" , above was "disapproving", which one will last?

Reply:
fixed